release-notes/docs/3.10.0.md

```diff
@@ -0,0 +1,149 @@
+//todo documentation link
+
+#### Privacy Policies
+
+App builders can now configure privacy policies for every language. When a privacy policy is configured for a language, users who load the app in this language are prompted to accept the policy, and are blocked from using the app until they have accepted. Privacy policies can be configured using `medic-conf` or from a new App Management page.    
+See [the documentation](https://docs.communityhealthtoolkit.org/apps/guides/security/privacy-policy/) for more information.  
+// todo picture?
+
+#### More granular replication depth configuration for offline users
+
+This release adds a new `replication_depth` configuration option that limits the depth of replicated reports. This allows app builders to configure roles that replicate deeper contacts, can report against these contacts, but will not download reports about these contacts that have been created by other users.  
+See [the documentation](https://docs.communityhealthtoolkit.org/apps/guides/performance/replication/#report-depth) for more information.
+
+#### Improved geolocation data
+
+3.10.0 improves the way geolocation information is obtained, from requesting an instant position to allowing up to 30 seconds to get a position lock. Additionally, the `geolocation` property is now updated every time a report is edited, and the all previous geolocation entries are saved in a log within the same report.  
+This change also supports the latest updates in `medic-android`, where Location permission is requested at runtime, when editing/creating a report, rather than demanded when starting the app for the first time.   
+
+#### Customizable header tabs icons
+
+As of 3.10.0, header tabs icons can be customized, allowing any [FontAwesome](https://fontawesome.com/v4.7.0/) icon or [resource icon](https://docs.communityhealthtoolkit.org/design/icons/) in SVG format to be used as a tab icon.
+Tabs icons can be configured using `medic-conf` or from a new App Management page.  
+See [the documentation](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/header_tabs/) for more information.  
+// todo picture? 
+
+#### Improved performance of processing deleted docs
+
+Because processing document deletes, specifically deleting read docs, is relatively time-consuming, increasing linearly by the number of users that exist on the instance, deletion processing is now split into: 
+- generating tombstones: no changes, tombstones are generated like before, within the Sentinel transition processing feed.
+- deleting read docs and info docs: done within a new Sentinel scheduled task that doesn't block the main transitions processing feed
+
+Additionally, Sentinel scheduled tasks are no longer executed sequentially, they are now executed in parallel, keeping track of which tasks are running. Every 5 minutes, every scheduled task that is not already running is started.
```

Comment:
Is there any way to show how much quicker this is?

Reply:
I don't think anybody did any kind of benchmarking. Would you like me to benchmark ? (I can use the exact numbers used in the issue description: 5000 deletes for 1000 users).

Reply:
If it's not too much work that'd be perfect.

Reply:
Oh man ... I lowered the number to 500 after estimating it would take 8 hours. I did the test with 5000 docs in the new version, tho.